src/App.tsx

```diff
@@ -27,6 +29,14 @@ function App() {
   const code = params.get("code")
   const isRedirect = (sessionId !== null || message !== null || code !== null)
 
+  // Handle RTL text direction for Arabic language and update the lang attribute
+  const { i18n: { language } } = useTranslation()
+  useEffect(() => {
+    const dir = language === 'ar' ? 'rtl' : 'ltr'
+    document.getElementsByTagName('html')[0].setAttribute("dir", dir);
+    document.getElementsByTagName('html')[0].setAttribute("lang", language.slice(-2));
```

Comment:
Used `.slice(-2)` here to grab the last two letters, so in the case of `sim_en` it will still be seen by the browser as `en`